File: Java/warehouse/common/src/main/java/com/wbs/common/database/base/model/WhereInfo.java
package com.wbs.common.database.base.model;

import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

/**
 * @author WBS
 * @date 2023/3/7 14:26
 * @desciption WhereInfo
 */
@Setter
@Getter
public class WhereInfo implements Serializable {

    /**
     * 字段
     */
    private String column;

    /**
     * 运算符：=、>、<、IN、LIKE、IS NULL
     */
    private String symbol;

    /**
     * 值
     */
    private Object value;

    /**
     * 操作符：AND、OR
     */
    private String operate;
}

File: IoT/Common/Common/Database/WhereInfo.cs
namespace Common.Database;

public class WhereInfo
{
    /// <summary>
    ///     字段
    /// </summary>
    public string Column { set; get; }

    /// <summary>
    ///     运算符：=、>、<、IN、 LIKE、 IS NULL
    /// </summary>
    public string Symbol { set; get; }

    /// <summary>
    ///     数据类型
    /// </summary>
    public string DataType { set; get; }

    /// <summary>
    ///     值
    /// </summary>
    public dynamic Val { set; get; }

    /// <summary>
    ///     操作符：AND、OR
    /// </summary>
    public string Operate { set; get; }
}